### ISSUE TITLE
Keyboard navigation inadequate for BE project collection list

### ISSUE THREAD

**Issue by dananjohnson** [ManifoldScholar/manifold#2208]
The sidebar list of project collections needs improvements to keyboard navigation:
- [x] currently a project collection can't be opened by keyboard
- [ ] ~visibility button loses focus when toggled~
- [x] DnD functionality doesn't work with keyboard. This is a more complex refactor potentially requiring a new DnD library. We'll revisit this at a later time.

**Comment by dananjohnson** [comment]
Loss of focus occurs on any interaction with the component, as a result of `List.Entities.List` being re-rendered.

**Comment by dananjohnson** [comment]
Progress resolving this issue is on `dj/2208`.

**Comment by zdavis** [comment]
@zdavis Needs to help with this.

**Comment by zdavis** [comment]
@bot bug

**Comment by zdavis** [comment]
Thank you for taking the time to report this bug. We've reviewed the issue and agree that this should be fixed. We’re adding a “planned” label to indicate that we consider this bug as part of our current work plan.

_This was an automated message, but please don't hesitate to reply. Our team watches these issues closely and will respond as soon as we're able to!_

**Comment by Santoz2** [comment]
Accessibility Triage Notes: UW's ATX team considers this a priority 1 issue, as the drag and drop button is completely unreachable. On top of this, loss of focus makes this very confusing.
